### ISSUE TITLE
Use ApiBehaviorOptions.ClientErrorMapping sample code problem

### ISSUE THREAD

**Issue by doggy8088** [dotnet/AspNetCore.Docs#15055]
https://docs.microsoft.com/en-us/aspnet/core/web-api/handle-errors?view=aspnetcore-2.2#use-apibehavioroptionsclienterrormapping

In the sample code of this section, the `options.SuppressMapClientErrors = true;` should assign `false` to it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3f6504cd-a059-c5d4-73fb-61222ef1c6e0
* Version Independent ID: 1b42352c-a34f-7703-aa3e-d7e3156bd0bd
* Content: [Handle errors in ASP.NET Core web APIs](https://docs.microsoft.com/en-us/aspnet/core/web-api/handle-errors?view=aspnetcore-2.2#use-apibehavioroptionsclienterrormapping)
* Content Source: [aspnetcore/web-api/handle-errors.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/web-api/handle-errors.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-webapi**
* GitHub Login: @pranavkm
* Microsoft Alias: **prkrishn**

**Comment by Rick-Anderson** [comment]
Use the `ClientErrorMapping` property to configure the contents of the `ProblemDetails` response. For example, the following code in `Startup.ConfigureServices` updates the `type` property for 404 responses:

```
services.AddMvc()
    .SetCompatibilityVersion(CompatibilityVersion.Version_2_2)
    .ConfigureApiBehaviorOptions(options =>
    {
        options.SuppressConsumesConstraintForFormFileParameters = true;
        options.SuppressInferBindingSourcesForParameters = true;
        options.SuppressModelStateInvalidFilter = true;
        options.SuppressMapClientErrors = true;    // <-- THIS SHOULD BE FALSE
        options.ClientErrorMapping[404].Link =
            "https://httpstatuses.com/404";
    });
```

**Comment by Rick-Anderson** [comment]
@pranavkm I pasted the relevant doc section above. Please review suggestion.

**Comment by Rick-Anderson** [comment]
Moved to #16385

**Comment by serpent5** [comment]
Fixed in #24066.
